includes/poly.php

```diff
@@ -9,55 +9,55 @@
 
 // Parameters
 
-$coords= explode('|',$_GET['coords']);
-$active = (isset($_GET['active'])) ? 1 : 0;
+$coords = explode( '|', $_GET['coords'] );
+$active = ( isset( $_GET['active'] ) ) ? 1 : 0;
 
-if (!$_GET['coords'] || count($coords)==0) {
-  die('No coordinates supplied');
+if ( !$_GET['coords'] || count( $coords ) == 0 ) {
+  die( 'No coordinates supplied' );
 }
 
 $maxx = 1;
 $maxy = 1;
 
-for ($i=0; $i<count($coords); $i+=2) {
+for ( $i = 0; $i < count( $coords ); $i += 2 ) {
   $x = $coords[$i];
-  $y = $coords[$i+1];
-  if ($x>$maxx) $maxx = $x;
-  if ($y>$maxy) $maxy = $y;
+  $y = $coords[$i + 1];
+  if ( $x > $maxx ) { $maxx = $x;
+  }
+  if ( $y > $maxy ) { $maxy = $y;
+  }
 }
 
-if ($maxx>2000) $maxx = 2000;
-if ($maxy>2000) $maxy = 2000;
+if ( $maxx > 2000 ) { $maxx = 2000;
+}
+if ( $maxy > 2000 ) { $maxy = 2000;
+}
```

Comment:
The if statements look weird.
Either it's a one-liner, in which case the } should be on the same line, or it's a block. Here it's a mixture.